tools/enhancements/metadata.go

```diff
@@ -43,3 +44,79 @@ func (s *Summarizer) GetMetaData(pr int) (*MetaData, error) {
 	}
 	return NewMetaData(content)
 }
+
+// Validate returns a list of issues found with the metadata
+func (m *MetaData) Validate() []string {
+	results := []string{}
+
+	reportError := func(msg string) {
+		results = append(results, msg)
+	}
+
+	// Must have one valid tracking link and no TBD
+	foundLink := false
+	for _, trackingLink := range m.TrackingLinks {
+		if trackingLink == "TBD" {
```

Comment:
We can probably ban `TBD` in all metadata properties, right?

Reply:
We could. I was focusing on the fields that seemed most important right now, based on issues I've seen with PRs.